templates/page/defaultSuggestion.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="no">
```

Comment:
Can you change the `lang` attribute to get the value injected through the template locale parameter?

Reply:
Done